domain/lib/src/model/sharedspacedocument/work_group_document.dart

```diff
@@ -77,3 +78,7 @@ class WorkGroupDocument extends WorkGroupNode {
     sha256sum,
   ];
 }
+
+extension WorkGroupDocumentExtension on WorkGroupDocument {
+  WorkGroupDocumentUti getWorkGroupDocumentUti() => WorkGroupDocumentUti(SupportedPreviewFileTypes.iOSSupportedTypes[mediaType.mimeType]);
```

Comment:
what happen if `SupportedPreviewFileTypes.iOSSupportedTypes[mediaType.mimeType]` return null?

Reply:
@manhduc-nguyen

Reply:
it will pass `null` to the library. In this case the library will get the `UTI` by itself. I did this on purposed.